README.md

```diff
@@ -504,14 +504,144 @@ Validates list input.
 
 # Template & Static files
 
+API Star is particularly designed around building APIs, but is equally
+capable of being used to build web applications.
+
 ## Templates
 
+To include templates in your application, create a `templates` directory
+alongside your `app.py` file, and use `app.render_template(template_name, **context)`.
+
+By default `jinja2` is used for template rendering.
+
+**templates/index.html**:
+
+```
+<html>
+    <body>
+        {% if name %}
+        <h1>Hello, {{ name }}!</h1>
+        {% else %}
+        <h1>Hello!</h1>
+        {% endif %}
+    </body>
+</html>
+```
+
+**app.py**:
+
+```
+from apistar import App, Route
+
+
+def welcome(app: App, name=None):
+    return app.render_template('index.html', name=name)
+
+
+routes = [
+    Route('/', method='GET', handler=welcome),
+]
+
+app = App(routes=routes)
+
+
+if __name__ == '__main__':
+    app.serve('127.0.0.1', 8080, use_debugger=True)
+```
+
+You can configure where templates are served from by using the `template_dir`
+argument when instantiating an application.
```

Comment:
As I understand `installed_packages` also can be used to control template search paths?